### PR TITLE
Fix threading design

### PR DESCRIPTION
**Pull request by GretaD**
- [x]  Fix the left space of threading(where the icon-mail starts)

- [x]  Fix the height on HTML messages

- [x] Fix timestamp and subject

- [x] remove space before comma

- [x] remove not active reply button

~~- [ ] add the non active reply button on the 3 dot menu~~

### CODE REVIEW

**Comment by GretaD** [comment]
@jancborchardt hi, can you please tell me which one is better from the options below. During the design call you proposed to remove the button from the not active ones and keep the active primary one. It looks more or less like this (1)

(1)
primarybutton

The (2) i moved the subject to the second line which in the future will be replaced by the first line of message. Design wise it looks better rather than having recipient and subject in one line.
(2)
newthreadindes

Instead of having (1) i would propose (3) since to me it looks more intuitive which message is active/open.
(3)
threadingnobutton

What do you think? (I am aware of the space problem at time stamp :) )

**Comment by jancborchardt** [comment]
Looks great @GretaD, very good proposal! :) We can even go a bit further than your Proposal 3 and
- For collapsed messages, put the "Reply" action into the 3-dot menu, so that the list gets less cluttered.
- Style the Reply button as `.primary` for one of the entries, for example for the last opened message? Cause otherwise we don’t have any primary action in the view. It’s good to have one though. :)

**Comment by GretaD** [comment]
> Looks great @GretaD, very good proposal! :) We can even go a bit further than your Proposal 3 and
> 
>     * For collapsed messages, put the "Reply" action into the 3-dot menu, so that the list gets less cluttered.
I like the idea to remove it, but i dont think its necessary to put it on 3dot menu, because i dont see any case scenario when someone would reply an un-open message. Google and hey doesnt even have the 3dot menu when the message its not open.
> 
>     * Style the Reply button as `.primary` for one of the entries, for example for the last opened message? Cause otherwise we don’t have any primary action in the view. It’s good to have one though. :)
The active/open/expanded one its still a primary button: https://github.com/nextcloud/mail/pull/3586/files#diff-d794712fba69cdd3a34a47f67fca32f5L43

i just removed the background color and the borders.

**Comment by jancborchardt** [comment]
> Google and hey doesnt even have the 3dot menu when the message its not open.

Alrighty, then let's just do the same. :) Simpler is better.

> i just removed the background color and the borders.

Yep yep – I mean though that 1 button should be the "blue action button", so that should probably be the one of the last opened one. That way it's very clear where the actionable button is.

**Comment by ChristophWurst** [comment]
> For collapsed messages, put the "Reply" action into the 3-dot menu, so that the list gets less cluttered.

- [ ] BUG: There is no "Reply" in that menu

**Comment by ChristophWurst** [comment]
Bildschirmfoto von 2020-09-21 14-24-12
Bildschirmfoto von 2020-09-21 14-21-30

IMO the positioning of the avatar is not ideal, the avatar and the two lines of text next to it should be one the same line or *vertically centered* if that explains it better.

Also long subjects go underneath the reply button

Bildschirmfoto von 2020-09-21 14-24-59

File: src/components/ThreadEnvelope.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 <template>
 	<div>
-		<div>
+		<div style="display: flex">
```
Comment (juliushaertl) [inline src/components/ThreadEnvelope.vue:24]: No inline please, just give the div a proper class

Comment (GretaD) [reply src/components/ThreadEnvelope.vue:24]: i didnt know how to call it. how about `leftandrigh` :smiley:

Comment (juliushaertl) [reply src/components/ThreadEnvelope.vue:24]: How about something like thread--header :)

File: src/components/AddressList.vue [hunk 2]
```diff
@@ -2,7 +2,7 @@
 	<span>
 		<template v-for="(entry, idx) in entries">
 			<Address :key="entry.email" :email="entry.email" :label="entry.label" /><!--
-			--><span v-if="idx === entries.length - 2" :key="'split' + entry.email">{{ t('mail', 'and') }}</span><!--
+			--><span v-if="idx === entries.length - 2" :key="'split' + entry.email">{{ t('mail', ' and') }}</span><!--
```
Comment (ChristophWurst) [inline src/components/AddressList.vue:5]: instead of this space, remove the html comment between the two `span`s.

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Rest looks fine now :)

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**